cypress/support/e2e.ts

```diff
@@ -1,5 +1,22 @@
 import "cypress-html-validate/commands";
 
 afterEach(() => {
-    cy.htmlvalidate();
+    cy.htmlvalidate({
+        rules: {
```

Comment:
Kanske lägga detta i [`cypress.config.ts`](https://github.com/Forsakringskassan/docs-generator/blob/main/cypress.config.ts#L8) där det är mer förväntat att hitta config.

Kommer inte alla som använder `docs-generator` behöva lägga in denna configen nu?

Reply:
✅